src/test/TestUtils/DotNetCli.cs

```diff
@@ -66,7 +66,7 @@ public Command Run(params string[] args)
                 .EnvironmentVariable("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "1");
         }
 
-        public Command Exec(string command, params string[] args)
+        private Command Exec(string command, params string[] args)
```

Comment:
Can we remove this and make all the below helpers use the one with just `string[]` args - this feels like almost pure code duplication.

Reply:
How? By duplicating the arg insertion code into all the clients?

Reply:
now that I understand params, guess I could just remove the changes to this file.